mrmoody-metrics/docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '2'
+services:
+  web:
+    image: "moody"
```

Comment:
this is an error - you probably have image moody build locally, that's why it works